bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -57,6 +57,7 @@ apiServerArguments:
     - "ServiceNodeExclusion=true"
     - "DownwardAPIHugePages=true"
     - "LegacyNodeRoleBehavior=false"
+    - "PodSecurity=true"
```

Comment:
@mfojtik @stlaz can you explain me what "bootstrap config overrides" is?

Reply:
(this change is required to pass unit tests)

Reply:
IIUC it's the overrides to default config that are specific to boostrapping